src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleErrorFieldSymbol.vb

```diff
@@ -22,42 +19,38 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         ''' </summary>
         Private ReadOnly _tupleElementIndex As Integer
 
-        Private ReadOnly _locations As ImmutableArray(Of Location)
         Private ReadOnly _useSiteDiagnosticInfo As DiagnosticInfo
-        Private ReadOnly _correspondingDefaultField As TupleErrorFieldSymbol
```

Comment:
I wouldn't remove those private fields. It is convenient to see what's in a type.